src/PrestaShopBundle/Controller/Admin/Improve/Design/CmsPageController.php

```diff
@@ -423,6 +432,13 @@ public function editCmsCategoryAction($cmsCategoryId, Request $request)
                 'cmsCategoryParentId' => $cmsCategoryParentId,
                 'enableSidebar' => true,
                 'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
+                'layoutTitle' => $this->trans(
+                    'Editing page category %name%',
```

Comment:
```suggestion
                    'Editing category %name%',
```

I don't know. I like the "Editing CMS category", and "Editing page category" is similar to "Edit category for this particular page". Maybe "pages category"...?

Reply:
Daniel's suggestion is clear and easy to scan but it needs a more specific translation domain.

Reply:
Decided to use page category